tests/e2e/performance/csi_tests/test_pvc_clone_performance.py

```diff
@@ -203,4 +251,80 @@ def test_clone_create_delete_performance(
+        average_deletion_time = statistics.mean(deletion_time_list)
+        average_deletion_speed = statistics.mean(deletion_speed_list)
+        logger.info(f"Average deletion time is  {average_deletion_time} secs.")
+        logger.info(f"Average deletion speed is  {average_deletion_speed} Mb/sec.")
         logger.info("test_clones_creation_performance finished successfully.")
+
+        self.results_path = get_full_test_logs_path(cname=self)
+        # Produce ES report
+        # Collecting environment information
+        self.get_env_info()
+
+        self.full_log_path = get_full_test_logs_path(cname=self)
+        self.results_path = get_full_test_logs_path(cname=self)
+        self.full_log_path += f"--PVC-SIZE-{pvc_size}"
+        logger.info(f"Logs file path name is : {self.full_log_path}")
+
+        # Initialize the results doc file.
+        full_results = self.init_full_results(
+            ResultsAnalyse(
+                self.uuid,
+                self.crd_data,
+                self.full_log_path,
+                "pvc_clone_performance",
+            )
+        )
+
+        full_results.add_key("interface", self.interface)
+        full_results.add_key("total_clone_number", max_num_of_clones)
+        full_results.add_key("pvc_size", self.pvc_size)
+        full_results.add_key("clone_creation_time", creation_time_list)
+        full_results.add_key("clone_deletion_time", deletion_time_list)
+        full_results.add_key("average_clone_creation_time", average_creation_time)
+        full_results.add_key("average_clone_deletion_time", average_deletion_time)
+        full_results.add_key("clone_creation_speed", creation_speed_list)
+        full_results.add_key("clone_deletion_speed", deletion_speed_list)
+        full_results.add_key("average_clone_creation_speed", average_creation_speed)
+        full_results.add_key("average_clone_deletion_speed", average_deletion_speed)
+
+        # Write the test results into the ES server
+        if full_results.es_write():
+            res_link = full_results.results_link()
+            logger.info(f"The Result can be found at : {res_link}")
+
+            # Create text file with results of all subtest (4 - according to the parameters)
+            self.write_result_to_file(res_link)
+
+    def test_pvc_clone_results(self):
+        """
+        This is not a test - it is only check that previous test ran and finish as expected
+        and reporting the full results (links in the ES) of previous tests (4)
+        """
+        self.number_of_tests = 8
+        self.results_path = get_full_test_logs_path(
+            cname=self, fname="test_clone_create_delete_performance"
+        )
+        self.results_file = os.path.join(self.results_path, "all_results.txt")
+        logger.info(f"Check results in {self.results_file}.")
+        self.check_tests_results()
+        self.push_to_dashboard(test_name="pvc clone performance")
```

Comment:
I'd name the test : "PVC Clone Performance"  with Caps lock - similar to the convention of other tables names.

Reply:
Thanks Yuli. I made that change